packages/new-deploy/src/bridge/BridgeContracts.ts

```diff
@@ -456,25 +491,6 @@ export default class BridgeContracts extends AbstractBridgeDeploy<config.EvmBrid
 
     // check verification addresses
     // TODO: add beacon and proxy where needed.
-    this.checkVerificationInput(
```

Comment:
the full verification input doesn't exist every time the deploy script runs. it only exists fully when you deploy everything fresh.

Reply:
as a higher-level comment, we need to ensure we are outputting the verification input, ideally in the same format which can be consumed by our existing verification code.

Reply:
https://github.com/nomad-xyz/monorepo/issues/103